Common/Securities/SecurityPortfolioManager.cs

```diff
@@ -534,20 +533,6 @@ public void SetAccountCurrency(string accountCurrency)
         {
             accountCurrency = accountCurrency.LazyToUpper();
 
-            // only allow setting account currency once
-            // we could try to set it twice when backtesting and the job packet specifies the initial CashAmount to use
-            if (_setAccountCurrencyWasCalled)
-            {
-                if (accountCurrency != CashBook.AccountCurrency)
-                {
-                    Log.Trace("SecurityPortfolioManager.SetAccountCurrency():" +
-                              $" account currency has already been set to {CashBook.AccountCurrency}." +
-                              $" Will ignore new value {accountCurrency}");
-                }
-                return;
-            }
-            _setAccountCurrencyWasCalled = true;
```

Comment:
We agreed on maintaining existing behavior, and for the cases where the brokerage does not know which is the account currency `AccountBaseCurrency` would be null, so the `BrokerageSetupHandler` won't set account current and the user algorithm can. Suggest we do this for the `PaperBrokerage : BacktestingBrokerage` so that paper user algorithms can test with other account currencies, since in any case the default account current for the algorithm is already `Currencies.USD`